vertx/src/main/java/com/microsoft/gctoolkit/vertx/jvm/AbstractJavaVirtualMachine.java

```diff
@@ -32,95 +29,112 @@
 
     private static final Logger LOGGER = Logger.getLogger(AbstractJavaVirtualMachine.class.getName());
 
-    private static final double LOG_FRAGMENT_THRESHOLD = 18000;
+    private static final double LOG_FRAGMENT_THRESHOLD = 60.0d;
 
-    private Diarizer jvmConfigurationFromParser;
-    private JvmConfiguration jvmConfigurationForCoreApi;
+    private Diary diary;
     private DateTimeStamp timeOfLastEvent;
     private final Map<Class<? extends Aggregation>, Aggregation> aggregatedData = new ConcurrentHashMap<>();
 
     @Override
     public boolean isG1GC() {
-        return jvmConfigurationFromParser.getDiary().isG1GC();
+        return diary.isG1GC();
     }
 
     @Override
     public boolean isZGC() {
-        return jvmConfigurationFromParser.getDiary().isZGC();
+        return diary.isZGC();
     }
 
     @Override
     public boolean isShenandoah() {
-        return jvmConfigurationFromParser.getDiary().isShenandoah();
+        return diary.isShenandoah();
     }
 
     @Override
     public boolean isParallel() {
-        return jvmConfigurationFromParser.getDiary().isParNew();
+        return diary.isParNew();
     }
 
     @Override
     public boolean isSerial() {
-        return jvmConfigurationFromParser.getDiary().isSerialFull();
+        return diary.isSerialFull();
     }
 
     @Override
     public boolean isCMS() {
-        return jvmConfigurationFromParser.getDiary().isCMS();
+        return diary.isCMS();
     }
 
     @Override
     public String getCommandLine() {
-        return jvmConfigurationFromParser.getCommandLine();
+        return ""; //todo: extract from diary... jvmConfigurationFromParser.getCommandLine();
```

Comment:
Does this need to be fixed before we merge?

Reply:
no, but it is a high priority todo...